docs/source/model_design/risk_factors.rst

```diff
@@ -203,6 +203,85 @@ Intermediates
 Effect Modification
 """""""""""""""""""
 
+A factor :math:`M` is said to be an effect modifier if the effect of the 
+exposure :math:`E` on disease :math:`D` varies for different values of 
+:math:`M`. Effect modification is sometimes also called *interaction*. We 
+illustrate this relationship below.
+
+.. figure:: effect_mod_arrow_diagram.svg
+  :align: center
+
+If :math:`M` is some dichotomous effect modifier, then :math:`B\neq C`, and to 
+understand the full story of how :math:`E` acts on :math:`O`, we cannot simply 
+consider :math:`A`.
```

Comment:
I think that we could expand upon the "we cannot simply consider *A*" statement and note that this might be a useful measure if we are looking at results for an aggregate model population (no sub-group analyses) that has similar prevalence of the effect modifier from the study population where A was measured. And then additionally note that we might want to use B and C rather than A in situations where we are interested in evaluating associations in specific subgroups OR in cases where we are evaluating model populations with variable levels of exposure to the effect modifier.

Reply:
thank you! that's a good idea and a highly relevant clarification